pkg/controller/dataimportcron-conditions.go

```diff
@@ -38,11 +38,7 @@ const (
 
 func updateDataImportCronCondition(cron *cdiv1.DataImportCron, conditionType cdiv1.DataImportCronConditionType, status corev1.ConditionStatus, message, reason string) {
 	if conditionType == cdiv1.DataImportCronUpToDate {
-		gaugeVal := float64(0)
-		if status != corev1.ConditionTrue {
-			gaugeVal = 1
-		}
-		DataImportCronOutdatedGauge.With(getPrometheusCronLabels(client.ObjectKeyFromObject(cron))).Set(gaugeVal)
+		metrics.SetDataImportCronOutdated(getPrometheusCronLabels(types.NamespacedName{Namespace: cron.Namespace, Name: cron.Name}), status != corev1.ConditionTrue)
```

Comment:
why not use ObjectKeyFromObject? and why not simply pass the ConditionStatus + rename to SetDataImportCronUpToDate?

Reply:
not sure if I understood the first question, but we get corn from this file so we need to lass it from this file, and no need to use `ObjectKeyFromObject` we can just pass directly the name and namespace like we did in all other operators. regarding `SetDataImportCronUpToDate` I don't agree, I like to leave as is since it's based on the metric name again like we did in all operators.

To sum it up I prefer to leave as is, if you strongly against please let me know.

Reply:
Sure, keep it as is.